I'm getting a build error on the branch:

2018-10-10T11:30:01.6679271Z ShipManager.cs(76,35): error CS7036: There is no argument given that corresponds to the required formal parameter 'setX' of 'Location.Location(decimal, decimal, decimal)' [/home/vsts/work/1/s/DeepSpace.Core/DeepSpace.Core.csproj]
2018-10-10T11:30:01.6963429Z 
2018-10-10T11:30:01.6964291Z Build FAILED.
2018-10-10T11:30:01.6965070Z 
2018-10-10T11:30:01.6965944Z ShipManager.cs(76,35): error CS7036: There is no argument given that corresponds to the required formal parameter 'setX' of 'Location.Location(decimal, decimal, decimal)' [/home/vsts/work/1/s/DeepSpace.Core/DeepSpace.Core.csproj]
2018-10-10T11:30:01.6966189Z     0 Warning(s)
2018-10-10T11:30:01.6966245Z     1 Error(s)
2018-10-10T11:30:01.6966279Z 
2018-10-10T11:30:01.6966358Z Time Elapsed 00:01:24.54
2018-10-10T11:30:03.8477839Z Build started, please wait...
2018-10-10T11:30:04.9716750Z ShipManager.cs(76,35): error CS7036: There is no argument given that corresponds to the required formal parameter 'setX' of 'Location.Location(decimal, decimal, decimal)' [/home/vsts/work/1/s/DeepSpace.Core/DeepSpace.Core.csproj]
2018-10-10T11:30:05.0401947Z ##[error]Bash exited with code '1'.
2018-10-10T11:30:05.0700825Z ##[section]Finishing: CmdLine

I suspect it's nothing major - probably just something trying to create something with the empty constructor. Can you resolve so I can merge in?

You should be able to do this by either changing the code creating the location or by adding defaults to the arguments eg:

Location(x = 0, y = 0, z = 0)
{

Thanks!

--- a/DeepSpace.Contracts/Location.cs
+++ b/DeepSpace.Contracts/Location.cs
@@ -6,7 +6,7 @@
         public decimal Y { get; set; }
         public decimal Z { get; set; }
 
-        public Location(decimal setX, decimal setY, decimal setZ) {
+        public Location(decimal setX = 0, decimal setY = 0, decimal setZ = 0) {
             X = setX;
             Y = setY;
             Z = setZ;

